package/gluon-offline-ssid/luasrc/usr/bin/gluon-offline-ssid

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/lua
+
+local unistd = require 'posix.unistd'
+local uci = require("simple-uci").cursor()
+local util = require 'gluon.util'
+local site = require 'gluon.site'
+
+-- -------------
+-- safety checks
+-- -------------
+
+-- TODO:
+-- pgrep -f autoupdater >/dev/null && safety_exit 'autoupdater running'
+
+function safety_exit(t)
+  io.write(t .. ", exiting with error code 2")
+  os.exit(2)
+end
+
+function shell_exec(command)
+  local file = assert(io.popen(command, 'r'))
+  local output = file:read('*all')
+  file:close()
+  return output
+end
+
+function file_exists(name)
+   local f=io.open(name,"r")
+   if f~=nil then io.close(f) return true else return false end
+end
+
+local UT = util.get_uptime()
+if UT < 60 then
+  safety_exit('less than one minute')
+end
+
+local phy
+uci:foreach('wireless', 'wifi-device', function(config)
+  phy = util.find_phy(config)
+  if not phy then
+    safety_exit('no hostapd-phy')
+  end
+end)
+
+function dump(o)
+   if type(o) == 'table' then
+      local s = '{ '
+      for k,v in pairs(o) do
+         if type(k) ~= 'number' then k = '"'..k..'"' end
+         s = s .. '['..k..'] = ' .. dump(v) .. ','
+      end
+      return s .. '} '
+   else
+      return tostring(o)
+   end
+end
+dump(phy); os.exit()
+
+-- only once every timeframe minutes the SSID will change to the Offline-SSID
+-- (set to 1 minute to change immediately every time the router gets offline)
+local MINUTES = uci:get('offline-ssid', 'settings', 'switch_timeframe') or 30
```

Comment:
You don't need to use capital letters for variable names and the config file should be called gluon-offline-ssid.

Reply:
1. decapitalization done ;)

2. not changed: other packages don't call their config files gluon-* either

Reply:
2. also fixed